Improved error handling for tinytga (#430)
jamwaffles/embedded-graphics: tinytga/CHANGELOG.md

### Changed

- **(breaking)** [#407](https://github.com/jamwaffles/embedded-graphics/pull/407) The `image_descriptor` in `TgaHeader` was replaced by `image_origin` and `alpha_channel_bits`.
- **(breaking)** [#420](https://github.com/jamwaffles/embedded-graphics/pull/420) To support the new embedded-graphics 0.7 image API a color type parameter was added to `Tga`.
- **(breaking)** [#430](https://github.com/jamwaffles/embedded-graphics/pull/430) The `graphics` feature was removed and the `embedded-graphics` dependency is now non optional.

> - **(breaking)** [#430](https://github.com/jamwaffles/embedded-graphics/pull/430) `Tga` does no longer implement `IntoIterator`. Pixel iterators can now be created using the `pixels` an `raw_pixels` methods.

Reviewer: ```suggestion
- **(breaking)** [#430](https://github.com/jamwaffles/embedded-graphics/pull/430) `Tga` no longer implements `IntoIterator`. Pixel iterators can now be created using the `pixels` and `raw_pixels` methods.
```
